### ISSUE TITLE
FlowExplorer: In SearchBar operators 'contains', '!contains' and 'startsWith' are not supported

### ISSUE THREAD

**Issue by bmukheja** [nuagenetworks/visualization-framework#789]
@nxanil - Flow explorer table(vss-enterprise-flow-explorer) filter box, the three search operators are missing. There is no configuration difference between this and say vss-domain-flow-table, yet the latter shows these three operators. Please have a look at the below screenshots. Let me know if you need any help in reproducing

![image](https://user-images.githubusercontent.com/15804045/49046318-8ab89900-f188-11e8-83f2-a747be87af53.png)

![image](https://user-images.githubusercontent.com/15804045/49046321-8f7d4d00-f188-11e8-9538-97712169f20c.png)

**Comment by natabal** [comment]
@nxanil Please add this to the items on your list for this week

**Comment by nxanil** [comment]
@natabal @bmukheja @ronakmshah 
As per my discussion with Ronak, for scroll searching where are only going to use: Equal and Not Equal to the operator, since I an generating the query dynamically.
But yes - I will keep this in my list, and look for the possible approach to add missing options for "Scroll" searching as well.

**Comment by natabal** [comment]
@nxanil I was not aware about this agreement! If this is the case then let's make sure that this is documented at least

**Comment by bmukheja** [comment]
Makes sense for scroll based queries, but we shouldn't confuse users then. I'll update the description of flow table to say it is fixed number of entries.
